rpc/common/moc_common_computecommon.proto

```diff
@@ -102,21 +102,23 @@ message WinRMConfiguration {
 	repeated WinRMListener Listeners = 1;
 }
 
-message Gpu {
-	enum AssignmentType {
-		GpuDDA = 0;
-		GpuP = 1;
-	}
+enum GpuAssignmentType {
```

Comment:
What impact does this have on the protobuf contract?

Reply:
For hybridaks, cloud operator will call cloud agent to get the cluster info (especially GPU Assign Type) to decide the plugins to install.

Reply:
@raghavendra-nataraj If you were asking about the change from AssignmentType to GpuAssignmentType, the AssignmentType enum is not being used anywhere, it was added by my earlier PR. So I don't think there should be any issue with protobuf contract

Reply:
When you say earlier. A release is been made with this protobuf. So I believe it might be impacted on the structure

Reply:
Oh got it, although the GPU struct is not being used in the new release, so it shouldn't matter, right? I spoke with Pradeep during office hours, and he believes it shouldn't impact either.

Pradeep brought a point that it may affect the marshaling of VM, but it is not very likely. I can test and verify this needed.

Regardless, if this feels like it could affect the existing structure, then we could keep using the same name for AssignmentType enum.

Reply:
Yes , lets test this scenario. We hit a similar issue in SDN where the we assumed that it as long as the field is not used. but we hit issues during marshalling

Reply:
I tested with the older release, and it seems there is some issue after replacing the binaries from this moc PR. We will revert the name change.